IoTCIntegration/lib/engine.js

```diff
@@ -52,10 +52,10 @@ module.exports = async function (context, device, measurements, timestamp) {
             message.properties.add('iothub-creation-time-utc', timestamp);
         }
 
-        await util.promisify(client.open.bind(client))();
```

Comment:
Please remove the import of `util` in line 10 now that it's not being used anywhere.